src/app/modules/group/pages/group-by-id/group-by-id.component.ts

```diff
@@ -43,8 +43,8 @@ export class GroupByIdComponent implements OnDestroy {
     this.subscriptions.push(
       this.groupDataSource.state$.pipe(
         readyData(),
-        map(({ group, route, breadcrumbs }): GroupInfo => groupInfo({
-          route,
+        map(({ group, route, breadcrumbs, navigation }): GroupInfo => groupInfo({
+          route: groupRoute(group, route.path),
```

Comment:
What's the use of recreating a group route ? Last time I saw `route`, it was of type `GroupRoute` already

Reply:
Was a mistake. Fixed.